eventlogger/src/main/java/gov/cms/ab2d/eventlogger/eventloggers/kinesis/KinesisEventProcessor.java

```diff
@@ -0,0 +1,60 @@
+package gov.cms.ab2d.eventlogger.eventloggers.kinesis;
+
+import com.amazonaws.ResponseMetadata;
+import com.amazonaws.services.kinesisfirehose.AmazonKinesisFirehose;
+import com.amazonaws.services.kinesisfirehose.model.PutRecordRequest;
+import com.amazonaws.services.kinesisfirehose.model.PutRecordResult;
+import com.amazonaws.services.kinesisfirehose.model.Record;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import gov.cms.ab2d.eventlogger.LoggableEvent;
+import lombok.extern.slf4j.Slf4j;
+
+import java.nio.ByteBuffer;
+import java.util.concurrent.Callable;
+
+@Slf4j
+public class KinesisEventProcessor implements Callable<Void> {
+    private AmazonKinesisFirehose client;
+    private String streamId;
+    private LoggableEvent event;
+
+    public KinesisEventProcessor(LoggableEvent event, AmazonKinesisFirehose client, String streamPrefix) {
+        this.client = client;
+        this.streamId = streamPrefix;
+        this.event = event;
+    }
+
+    static String getJsonString(LoggableEvent event) throws JsonProcessingException {
+        ObjectMapper mapper = new ObjectMapper()
```

Comment:
ObjectMapper is threadsafe, you can declare it as a private static field https://stackoverflow.com/questions/3907929/should-i-declare-jacksons-objectmapper-as-a-static-field

Reply:
Good to know, thanks.